guide/microbot.md

```diff
@@ -0,0 +1,20 @@
+## PICAXE Microbot
+
+%figure "Mircobot model in Webots"
+
+![model.png](images/robots/microbot/model.png)
+
+%end
+
+The [PICAXE microbot system](http://www.picaxe.com/Hardware/Robot-Kits/PICAXE-20X2-Microbot/) allows to create a robot from hardware pieces.
+The system can be fully customised by the end user, with the capacity to customize via different input sensors and output devices (in addition to the on-board motor/gearbox, LED and piezo outputs).
```

Comment:
customised &rarr; customized
replace ", with the capacity to customize via" with " using"